arc/scenarios/generator.py

```
@@ -76,93 +78,60 @@ async def generate_scenarios_batch(
         self,
         count: int = 100,
         batch_size: int = 20,
-        pattern_ratio: float = 0.7
+        pattern_ratio: float = 0.7,
+        currency_focus: bool = False
     ) -> List[Scenario]:
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)